en/Obsidian/Community code of conduct.md

```diff
@@ -2,24 +2,40 @@
 
 These are the behaviors we do not tolerate.
 
-### Major offenses
+### Serious offenses
+
+Once verified, these offenses will result in an immediate ban from our community.
 
 - **Harassment**: including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion, as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
 - **Threats**: threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.
 
-### Minor offenses and other behaviors
+### Other offenses
+
+Once verified, these offenses will result in a warning. On second offence, an immediate ban will be issued.
```

Comment:
Rewrite for active voice, Canadian spelling of "offences", and to offer more flexibility on enforcement:

> Users who commit these offences will be sent a warning. Repeated offences will result in the user being banned from the community.